generic-ec-core/src/lib.rs

```diff
@@ -248,3 +248,12 @@ pub trait Reduce<const N: usize> {
     /// integer modulo curve (prime) order
     fn from_le_array_mod_order(bytes: &[u8; N]) -> Self;
 }
+
+/// Marker trait for curves whose underlying implementation doesn't allow
+/// representing invalid points.
+/// # Safety
+/// Safe to implement when the checks for invalid points always return `true`.
+/// Those checks are:
```

Comment:
You mean check for valid points? :thinking: 

If checks for invalid points return `true`, then it's an invalid point?